CONTRIBUTION.md

```diff
@@ -0,0 +1,137 @@
+
+Once you have your code ready for review, please  [open a pull request][pull-requests]. Please follow
+the [pull request guidelines][pr-guide]. If you are new to the project, the 
+[new contributors][new-contributors] section provides useful information to get you started. A 
+committer of the Trace Extension will then review your contribution and help to get it merged.
+
+Please note that before your pull request can be accepted, you must electronically sign the 
+[Eclipse Contributor Agreement][eca]. For more information, see the 
+[Eclipse Foundation Project Handbook][commiter-handbook].
+
+### Pull request guidelines 
+
+**Changes to the project** are made by submitting code with a pull request (PR).
+
+* [How to write and submit changes][creating-changes]
+* [Example pull request][issue-402]
+
+**Good commit messages** make it easier to review code and understand why the changes were made. 
+Please include a:
+
+* *Title:* Concise and complete title written in imperative (e.g. "Update Gitpod demo screenshots"
+or "Single-click to select or open trace")
+* *Problem:* What is the situation that needs to be resolved? Why does the problem need fixing? 
+Link to related issues (e.g. "Fixes [#317][issue-317]").
+* *Solution:* What changes were made to resolve the situation? Why are these changes the right fix?
+* *Impact:* What impact do these changes have? (e.g. Numbers to show a performance improvement, 
+screenshots or a video for a UI change)
+* [*Sign-off:*][sign-off] Use your full name and a long-term email address. This certifies that you 
+have written the code and that, from a licensing perspective, the code is appropriate for use in open-source.
+
+Other commit information:
+
+* [How to format the message][commit-message-message]
+* [Example commit message][commit-message-example]
+
+### New contributors
+
+#### Explore the project context
+
+* **Check out the [tracevizlabs][tracevizlab].** You don't need to complete all of them, but reading
+ a couple is good for context. Lab sets 0 and 1 are especially useful.
+* **Browse recent conference presentations.** They give an overview of the trace viewer's goals, 
+[architecture][architecture], and functionality.
+  * *Trace Compass Cloud: Eclipse Trace Compass's migration towards Theia*, [slides][tracompa-cloud-slides], 
+  [video][tracompa-cloud-video] - Bernd Hufmann, Ericsson AB (EclipseCon 2021)
+  * *Using Theia to take trace analysis and visualization to the next level*, [slides][tracing-with-theia-slides], 
+  [video][tracing-with-theia-video] - Bernd Hufmann, Ericsson AB (EclipseCon 2019)
+  * *A New Flexible Architecture for Trace Compass*, [slides][flexible-architecture-slides], 
+  [video][flexible-architecture-video] - Michel Dagenais, Polytechnique Montréal ([Tracing Summit 2019][tracing-summit])
+* **Get an [overview of external components][external-components]** used by this trace viewer. See how they interact.
+
+#### Start with a small change
```

Comment:
Since the other chapters are move up, have this chapter as heading 3: ### Start with a small change